### DISCUSSION TITLE
Multiple printers access from 1 orca slicer instance.

### DISCUSSION THREAD

**Discussion by dweenz** [OrcaSlicer/OrcaSlicer#7506] [Dev]
My idea is to have access to all my printers from one instance of Orca. When you add a plate you would be able to designate each plate their own printer. Then depending which plate you slice and send print from could open the web gui in device tab. Which if multiple are open could have tabs to select between each printers web gui. I know most people are fine with opening multiple instances but I feel this could be a move forward for productivity and ease of multipart and color prints transferring between printers work loads.

**Comment by Mingasux** [comment]
This would be brilliant to have!  Slicing could be slow on "All plates" I reckon but be well worth it

**Comment by MrBill57** [comment]
I fight this all the time. Work on 3 or more different projects for 3 different printers from ONE instance of Orca.   This has brought up issues to fix many times. I'm looking at what to do to run multiple instances, but the ability to assign a plate to a printer, and bring up the profile, etc., associated with that plate, would be a great thing. Sounds like a lot of work, to be honest. But what utility!
